MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -536,6 +536,9 @@ - (MUXSDKPlayerData *)getPlayerData {
         }
     }
     // TODO: Airplay - don't set the view if we don't actually know what is going on.
```

Comment:
Not sure what this `TODO` means 🤔

Reply:
You know what, I'm not sure either... at this point, your guess is as good as mine right now